library/.eslintrc

```diff
@@ -126,64 +178,53 @@
     "no-useless-concat": "warn",
     "no-useless-constructor": "warn",
     "no-useless-escape": "warn",
-    "no-useless-rename": [
-      "warn",
-      {
-        "ignoreDestructuring": false,
-        "ignoreImport": false,
-        "ignoreExport": false
-      },
-    ],
-    "no-with": "warn",
     "no-whitespace-before-property": "warn",
+    "no-with": "warn",
     "radix": "warn",
     "require-yield": "warn",
     "rest-spread-spacing": ["warn", "never"],
     "strict": ["warn", "never"],
     "unicode-bom": ["warn", "never"],
     "use-isnan": "warn",
     "valid-typeof": "warn",
-    "no-restricted-properties": [
-      "error",
-      // TODO: reenable once import() is no longer slow.
-      // https://github.com/facebookincubator/create-react-app/issues/2176
-      // {
-      //   object: "require",
-      //   property: "ensure",
-      //   message: "Please use import() instead. More info: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#code-splitting"
-      // },
-      {
-        "object": "System",
-        "property": "import",
-        "message": "Please use import() instead. More info: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#code-splitting"
-      },
-    ],
 
     // https://github.com/benmosher/eslint-plugin-import/tree/master/docs/rules
-    "import/first": "error",
-    "import/no-amd": "error",
-    "import/no-webpack-loader-syntax": "error",
+    "import/first": "warn",
+    "import/no-amd": "warn",
+    "import/no-webpack-loader-syntax": "warn",
+    "import/no-duplicates": "warn",
+    "import/export": "warn",
+    "import/default": "warn",
+    // "import/no-unresolved": "warn", // we should write our own resolver https://github.com/benmosher/eslint-plugin-import#resolvers
```

Comment:
@EECOLOR is it possible to write a custom resolver for `@kaliber/build`?
- https://github.com/benmosher/eslint-plugin-import#resolvers
- https://github.com/benmosher/eslint-plugin-import/tree/master/resolvers

Reply:
Sure thing, we should create a separate issue to add this.

https://github.com/benmosher/eslint-plugin-import/blob/master/resolvers/README.md